src/iap/shared/components/carousel/carousel.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="carousel">
+	<div class="carousel__slide-container">
+		<ng-template slideHost></ng-template>
+	</div>
+
+	<div
+		class="carousel__prev-button"
+		(click)="onPrevButtonClick()"
+	>
+		<span class="carousel__prev-button-label"><</span>
+	</div>
+
+	<div
+		class="carousel__next-button"
+		(click)="onNextButtonClick()"
+	>
+		<span class="carousel__next-button-label">></span>
+	</div>
+
+	<ul class="carousel__pages">
+		<li
+			*ngFor="let slide of slides; let i = index"
+			class="carousel__pages-button"
+			[ngClass]="{ 'carousel__pages-button_selected': currentSlideIndex === i }"
```

Comment:
It's a good practice to move all the logic from templates to components.
```suggestion
			[class.carousel__pages-button_selected]="isCurrentSlide(i)"
```
P.S. You should implement `isCurrentSlide` method yourself. :smiley: